wemake_python_styleguide/visitors/ast/complexity/overuses.py

```diff
@@ -48,13 +48,15 @@ class StringOveruseVisitor(base.BaseNodeVisitor):
         '\n',
         '\r\n',
         '\t',
+        '|',
         b' ',
         b'.',
         b',',
         b'',
         b'\n',
         b'\r\n',
         b'\t',
+        b'|',
```

Comment:
I don't think we should add this one. Annotations are always strings.

Reply:
Ok